include/range/v3/view/linear_distribute.hpp

```diff
@@ -71,15 +75,16 @@ namespace ranges
                 }
                 else
                 {
-                    from_ += (to_ - from_) / T(n_);
+                    from_ = T(to_ - (delta_ * (n_ - 1)));
                 }
             }
 
         public:
             constexpr linear_distribute_view() = default;
-            constexpr linear_distribute_view(T from, T to__, std::ptrdiff_t n) noexcept
+            constexpr linear_distribute_view(T from, T to, std::ptrdiff_t n) noexcept
               : from_(from)
-              , to_(to__)
+              , to_(to)
+              , delta_((to - from) / Calc(n - 1))
```

Comment:
division by 0 if `n == 1`

Reply:
You are right, I've missed it. It didn't affect the functionality as for n == 1 the delta was not used in calculations.